kahuna/app/views/main.scala.html

```diff
@@ -43,7 +44,8 @@
           feedbackFormLink: "@Html(feedbackFormLink.getOrElse(""))",
           usageRightsHelpLink: "@Html(usageRightsHelpLink.getOrElse(""))",
           invalidSessionHelpLink: "@Html(invalidSessionHelpLink.getOrElse(""))",
-          supportEmail: "@Html(supportEmail.getOrElse(""))"
+          supportEmail: "@Html(supportEmail.getOrElse(""))",
+          fileMetadataConfig: @Html(fileMetadataConfigs)
```

Comment:
I'm somewhat apprehensive of having a field here that isn't escaped but it looks like we have a number of other examples and it isn't coming from the user so it's relatively safe.